### PR TITLE
create complete graph function

### PR DESCRIPTION
**Pull request by susano0**
The query works as follows: 
image
image

### CODE REVIEW

**Comment by susano0** [comment]
test expr                         ... FAILED

Need to fix the regression test in expr (issue already exists #306)

**Comment by JoshInnis** [comment]
You need to research why the regression tests are failing. A change you made is not compatible with the system

**Comment by susano0** [comment]
Fixed the issue

File: src/backend/utils/graph_generation.c [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+    graph_name_str = NameStr(*graph_name);
+    vtx_name_str = AG_DEFAULT_LABEL_VERTEX;
+    edge_name_str = NameStr(*edge_label_name);
+
+
+    if (!graph_exists(graph_name_str))
+    {
+        DirectFunctionCall1(create_graph, CStringGetDatum(graph_name));
+
```
Comment (JoshInnis) [inline src/backend/utils/graph_generation.c:128]: extra newline

**Comment by JoshInnis** [comment]
Please add regression tests to this and fix the age--1.1.0.sql file

**Pull request merged**